### PR TITLE
chore: Migrate to SPDX headers

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

File: lib/Capabilities.php [hunk 1]
```diff
@@ -2,26 +2,8 @@
...
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- *
+ * SPDX-FileCopyrightText: 2018 Joas Schilling <coding@schilljs.com>
```
Comment (AndyScherzinger) [inline lib/Capabilities.php:5]: Lost Kate copyright line

File: lib/MatterbridgeManager.php [hunk 2]
```diff
@@ -2,25 +2,8 @@
...
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- *
+ * SPDX-FileCopyrightText: 2020 Julien Veyssier <eneiluj@posteo.net>
```
Comment (AndyScherzinger) [inline lib/MatterbridgeManager.php:5]: Lost Kate copyright line

File: lib/Service/PollService.php [hunk 3]
```diff
@@ -2,25 +2,8 @@
...
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- *
+ * SPDX-FileCopyrightText: 2023 Joas Schilling <coding@schilljs.com>
```
Comment (AndyScherzinger) [inline lib/Service/PollService.php:5]: 2022 in the original

File: lib/CachePrefix.php [hunk 4]
```diff
@@ -1,29 +1,12 @@
 <?php
-/*
- * @copyright Copyright (c) 2024 Joas Schilling <coding@schilljs.com>
+/**
+ * SPDX-FileCopyrightText: 2024 Joas Schilling <coding@schilljs.com>
```
Comment (Antreesy) [inline lib/CachePrefix.php:3]: Not a duplicate for this file?

File: lib/Capabilities.php (hunk 1 above)
Comment (nickvergessen) [reply lib/Capabilities.php:5]: She was only author, not a copyright... Not sure how we'd handle this in server.

File: lib/CachePrefix.php (hunk 4 above)
Comment (nickvergessen) [reply lib/CachePrefix.php:3]: Yeah file was borked before scripting

File: lib/Capabilities.php (hunk 1 above)
Comment (AndyScherzinger) [reply lib/Capabilities.php]: Basically the same thing in our case: https://reuse.software/faq/#copyright-holder-author

File: lib/CachePrefix.php (hunk 4 above)
Comment (AndyScherzinger) [reply lib/CachePrefix.php]: I fixed it 👍

**Review by Antreesy — APPROVED** [review]
There's around 60 files without a creation year in copyright header. But given amount of changes, I guess, we are good for now, that's something could be added for later, if CI is green now

**Comment by Antreesy** [comment]
UPD: here are missing years: https://github.com/nextcloud/spreed/commit/24ad3a635570cbfa99d7bfb9a198aaad0889f60c

```sh
directory="."
substring="SPDX-FileCopyrightText: Nextcloud"

# Find files recursively in the directory
find "$directory" -type f -exec grep -l "$substring" {} + | while IFS= read -r file
do
    # Get the creation year of the file
    creation_year=$(git log --diff-filter=A --follow --format=%aI "$file" | tail -n 1 | cut -d'-' -f1)
    echo "$creation_year"

    # Insert the creation year at the specified place in the file
    sed -i "s~$substring~SPDX-FileCopyrightText: $creation_year Nextcloud~g" "$file"
done
```

**Review by AndyScherzinger — APPROVED** [review]

**Pull request merged**